### ISSUE TITLE
Resizing image process is unstable

### ISSUE THREAD

**Issue by nhuhoa** [fiji/register_virtual_stack_slices#7]
Dear Ignacio Arganda-Carreras,

Thanks for very nice work. My work need to process big volume images and have tested several image registration plugins.  And your plugins runs very well, fast execution time, multi-threads, good output. 
However, when I tested your plugin, I have noticed a small issue that the outputs are not stable. I think the problem maybe is at resizing image step cause I did not have time to go through the code. 
For my tests: 
+ First I use dapi images as references. Based on that I get the transformation maps (.xml files)
Size of raw dapi images: 13850 x 20544, registered images: 14080 x 20715 pixels
+ Then I apply the transformation maps .xml to process different channels (markers). 
    - First test:  marker images: 13850 x 20544, registered images: 14079 x 20715 pixels. The results are not    what I expect, the width of registered images is less 1px. 
    - Second test: marker images: 13850 x 20544, registered images: 14080 x 20715 pixels. The results are as expected. The size of marker images are the same as reference images. 

So I note the issue here for you. 
Thanks a lot, 
Hoa Tran

**Comment by iarganda** [comment]
Hello @nhuhoa and thanks for reporting! Could you upload somewhere the XML files of a couple of your images so I can try to reproduce the error? Thanks a lot!

**Comment by nhuhoa** [comment]
Dear Ignacio Arganda-Carreras,
Thanks for very quick reply. I send you the .xml file in the following link. Can you help me to look at it? I send 3 .xml files that I used, all produces same error. 
https://drive.google.com/open?id=1ECmZAkOpuH8dkVGdTZcdmduRQyRgXN7E
If in case you still can not detect any error, I will try to prepare some small images, test it and send you the images + xml files. 
Thanks a lot, 
Hoa Tran

**Comment by iarganda** [comment]
Thanks a lot, @nhuhoa . I have tried with your .xml files and I got the expected image sizes. Can you check if the pixel size is different on the images that gave you the problem?

**Comment by nhuhoa** [comment]
Dear Ignacio Arganda-Carreras, 
I think you figure out the correct answer. I tested 3 images and find the mapping: reference image & round 1 image, reference image & round 2 image. I rescaled the image of round 2 with the ratio of 0.97 before doing registration process. I guess this is the reason of unstable result. I will test it again without rescaling image. 
Thanks a lot for your support. 
Best, 
Hoa Tran
